content/en/docs/reference/kubernetes-api/common-parameters/common-parameters.md

```diff
@@ -58,6 +58,35 @@ When present, indicates that modifications should not be persisted. An invalid o
 
 
 
+## fieldValidation {#fieldValidation}
+
+When both the fieldValidation parameter is present and the `ServerSideFieldValidation` feature gate is enabeld, it instructs the server on how to handle objects in the request (POST/PUT/PATCH) containing unknown or duplicate fields. Valid values are:
+- Ignore: This will ignore any unknown fields, silently dropping them from the
+  object, and ignoring all but the last duplicate field that the decoder
+  encounters. This is the default behavior prior to v1.23 and is the default
+  behavior when the `ServerSideFieldValidation` feature gate is disabled.
+- Warn: This will send a warning via the [standarding warning response
+  header](https://datatracker.ietf.org/doc/html/rfc7234#section-5.5) for each
+  unknown or duplicate field it encounters. The request will still succeed if
+  there are no other errors, the resulting object will drop any unknown
+  fields and only persist the last of any duplicate fields. This is the default
+  when the `ServerSideFieldValidation` feature gate is enabled.
+- Strict: This will fail the request with a BadRequest error if any unknown or
+  duplicate fields are present. The error returned from the server will contain
```

Comment:
```suggestion
- Strict: This will fail the request with a BadRequest error if any unknown fields
  would be dropped from the object, or if any duplicate fields are present. The error returned from the server will contain
```